### ISSUE TITLE
Create SP's do not set success indicator

### ISSUE THREAD

**Issue by sjohansson** [data-solution-automation-engine/DIRECT#34]
the stored procedures should set the success indicator on successful completion, so that the caller knows the communicated status from the call.
These are lost in some SP's.
Add them and return success on successful completion

**Comment by RoelantVos** [comment]
Yes, that's a nice addition. There is no material impact (the processes work as expected at the moment) but it is definitely a nice polish and will help clarify what's going on.
